README.md

```diff
@@ -1,18 +1,38 @@
 # Managed Structure Template
-This template is a starting point for creating programs that can run as Griptape Cloud Managed Structures. 
+
+This template is a starting point for creating programs that can run as Griptape Cloud Managed Structures.
 It provides a basic project layout for creating a Managed Structure, but you can customize it to fit your needs.
 
 See the [Griptape CLI](https://github.com/griptape-ai/griptape-cli?tab=readme-ov-file#skatepark-emulator) instructions for how to get started with this template.
 
 ## Project Structure
+
 The template provides the following project layout:
 
 - `structure.py` Contains the Managed Structure code.
 - `requirements.txt` Contains the dependencies for the Managed Structure. These will be automatically installed when the Structure is registered with Skatepark.
-- `example-client/client.py` Contains an example client that _uses_ the Managed Structure's API. This is useful for testing your Managed Structure locally but ultimately you will want to integrate your Managed Structure with your own application. 
+- `structure_file.yaml` Contains the Managed Structure configuration required by Skatepark and Griptape Cloud to properly build and run the Structure.
+- `example-client/client.py` Contains an example client that _uses_ the Managed Structure's API. This is useful for testing your Managed Structure locally but ultimately you will want to integrate your Managed Structure with your own application.
 - `example-client/pyproject.toml` Contains the dependencies for the example client. Poetry is only used for the example client and is not used by the Managed Structure itself.
 
+### Structure File Contents
```

Comment:
per earlier, can we change to `Structure Config` or `Structure Configuration` throughout?